modules/storage-persistent-storage-pv.adoc

```diff
@@ -179,8 +179,10 @@ endif::[]
 
 3. Use a recreate deployment strategy for pods that rely on AWS EBS.
 
+4. Only Raw block volumes can have ReadWriteMany access mode. For more information, see "Block volume support".
```

Comment:
I think mentioning Fibre Channel and iSCSI here is better, and suggest using "support" instead of "have":
For Fibre Channel and iSCSI, only Raw block volumes support ReadWriteMany access mode. For more information, see "Block volume support". 
(Can we add a link for Block volume support)

Reply:
Link cannot be added as it is a module.